packages/terra-application-links/src/terra-dev-site/test/application-links/ApplicationTabsLengthChange.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import Base from 'terra-base';
+import ApplicationTabs from '../../../tabs/ApplicationTabs';
+import testShortConfig from '../common/testShortConfig';
+import testLinkConfig from '../common/testLinkConfig';
+import messages from '../../../../translations/en-US.json';
+
+export default class extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleToggle = this.handleToggle.bind(this);
+    this.state = {
+      shortLinks: true,
+    };
+  }
+
+  handleToggle() {
+    this.setState(prevState => ({ shortLinks: !prevState.shortLinks }));
+  }
+
+  render() {
+    return (
+      <Base locale="en-US" customMessages={messages}>
```

Comment:
The test page is already wrapped in Base. By adding it here, its nesting i18n-provides and preventing translations switching for this test page. Is it possible to remove it from this example?

Reply:
Update: https://github.com/cerner/terra-framework/pull/365/commits/98ceed58574d727fa9a2044985bc6b348873c529